site/js/modules/webui/course.js

```diff
@@ -49,7 +50,10 @@ function handlerCourse(path, params, context, container) {
 
     container.innerHTML = `
         <h2>${course.name}</h2>
-        ${Render.cards(cards)}
+        <h3>Assignments</h3>
+        ${Render.cards(assignmentCards)}
+        <h3>Actions</h3>
+        ${Render.cards(actionCards)}
```

Comment:
These should probably be in different divs so they can be rendered with correct spacing.